eclipse_plugin/pom.xml

```diff
@@ -0,0 +1,103 @@
+<!--
+  ~ Copyright 2015 Google Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.google.googlejavaformat</groupId>
+    <artifactId>google-java-format-parent</artifactId>
+    <version>1.3-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>google-java-format-eclipse-plugin</artifactId>
+  <version>0.0.1</version>
+  <packaging>eclipse-plugin</packaging>
+  <name>Google Java Format Plugin for Eclipse 4.5+</name>
+
+  <description>
+    A Java source code formatter that follows Google Java Style.
+  </description>
+
+  <properties>
+    <tycho-version>0.26.0</tycho-version>
+  </properties>
+
+  <repositories>
+    <repository>
+      <id>mars</id>
+      <layout>p2</layout>
+      <url>http://download.eclipse.org/releases/mars</url>
+    </repository>
+  </repositories>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.google.googlejavaformat</groupId>
+      <artifactId>google-java-format</artifactId>
+      <version>1.3-SNAPSHOT</version>
```

Comment:
Resolves to locally defined version `0.0.1`.

Having a global property like:
```
  <properties>
    <googlejavaformat.version>1.3-SNAPSHOT</googlejavaformat.version>
  </properties>
```
seemed to help, but:
```
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for com.google.googlejavaformat:google-java-format:jar:1.3-SNAPSHOT
[WARNING] 'version' contains an expression but should be a constant. @ com.google.googlejavaformat:google-java-format-parent:${googlejavaformat.version}, ...\sormuras\google-java-format\pom.xml, line 39, column 12
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for com.google.googlejavaformat:google-java-format-eclipse-plugin:eclipse-plugin:1.0-${format.version}
[WARNING] 'version' contains an expression but should be a constant. @ line 27, column 12
[WARNING] 'version' contains an expression but should be a constant. @ com.google.googlejavaformat:google-java-format-parent:${googlejavaformat.version}, ...\sormuras\google-java-format\pom.xml, line 39, column 12
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for com.google.googlejavaformat:google-java-format-parent:pom:1.3-SNAPSHOT
[WARNING] 'version' contains an expression but should be a constant. @ com.google.googlejavaformat:google-java-format-parent:${googlejavaformat.version}, ...\sormuras\google-java-format\pom.xml, line 39, column 12
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
```
Gonna leave it constant.